### DISCUSSION TITLE
Window borders

### DISCUSSION THREAD

**Discussion by adammfurman** [ianyh/Amethyst#1639] [Ideas]
When shifting focus between windows, it is difficulty to discern the currently focused window. Colored window borders help solve this issue for me. 

Feature request:
Ability to set colored window borders separately for focused and non-focused windows. 

How I solve today: Use a separate tool for this sole purpose. 

Priority: Low, as another tool solves this issue, but is a nice to consolidate.

**Comment by ianyh** [comment]
This is a common request. I have been hesitant to introduce it because it's basically brand new functionality, and I have limited time to maintain existing features. It's kind of like the binary space partitioning layout; it's a user requested feature, but it's a bit half baked on my part because there is a lot of impedance mismatch, which means it has kind of fallen by the wayside.

**Comment by lakerfan0306** [comment]
> When shifting focus between windows, it is difficulty to discern the currently focused window. Colored window borders help solve this issue for me.
> 
> Feature request: Ability to set colored window borders separately for focused and non-focused windows.
> 
> How I solve today: Use a separate tool for this sole purpose.
> 
> Priority: Low, as another tool solves this issue, but is a nice to consolidate.



> When shifting focus between windows, it is difficulty to discern the currently focused window. Colored window borders help solve this issue for me.
> 
> Feature request: Ability to set colored window borders separately for focused and non-focused windows.
> 
> How I solve today: Use a separate tool for this sole purpose.
> 
> Priority: Low, as another tool solves this issue, but is a nice to consolidate.

What tool do you use? Been trying to find a way to do this as well

**Reply by adammfurman** [reply]
"borders", it's in the homebrew repository @lakerfan0306
